examples/seismic/model.py

```diff
@@ -490,22 +502,36 @@ def domain_size(self):
 class Model(GenericModel):
     """The physical model used in seismic inversion processes.
 
-    :param origin: Origin of the model in m as a tuple in (x,y,z) order
-    :param spacing: Grid size in m as a Tuple in (x,y,z) order
-    :param shape: Number of grid points size in (x,y,z) order
-    :param space_order: Order of the spatial stencil discretisation
-    :param vp: Velocity in km/s
-    :param nbpml: The number of PML layers for boundary damping
-    :param epsilon: Thomsen epsilon parameter (0<epsilon<1)
-    :param delta: Thomsen delta parameter (0<delta<1), delta<epsilon
-    :param theta: Tilt angle in radian
-    :param phi: Asymuth angle in radian
-
-    The :class:`Model` provides two symbolic data objects for the
+    Parameters
+    ----------
+    origin :
+        Origin of the model in m as a tuple in (x,y,z) order
+    spacing :
+        Grid size in m as a Tuple in (x,y,z) order
+    shape : int
+        Number of grid points size in (x,y,z) order
+    space_order : int
+        Order of the spatial stencil discretisation
+    vp : float
+        Velocity in km/s
+    nbpml : int
+        The number of PML layers for boundary damping
+    epsilon : float
+        Thomsen epsilon parameter (0<epsilon<1)
+    delta : float
+        Thomsen delta parameter (0<delta<1), delta<epsilon
+    theta : float
+        Tilt angle in radian
+    phi : float
+        Asymuth angle in radian
```

Comment:
missing ``subdomains`` parameter here.
Its datatype is "SubDomain, optional" . A possible description is "The Grid sub-region in which the wave is propagated (does not include boundaries)." @mloubout feedback?